portfolio/src/main/java/com/google/sps/classes/Comment.java

```diff
@@ -25,6 +29,14 @@ public long getTimeStamp() {
         return timestamp;
     }
 
+    public double getSentiment() {
+        return sentiment;
+    }
```

Comment:
Is this value used anywhere on the frontend? If not, it can probably be removed.

Reply:
I was thinking that maybe, if in the future I'd like to create an admin dashboard to display the analytics and the results of the comments, it would be better to store the numeric value of the sentiment rather than the sole tag.

Reply:
That makes sense, but there's a common mantra in agile software development: [YAGNI - You Aren't Gonna Need It](https://www.martinfowler.com/bliki/Yagni.html). Basically the idea is to defer adding fields/methods/features until you have a real use-case for them. It this case, adding a new field to the `Comment` class is a backwards-compatible change, so you can add it at any point in the future with little risk of breaking anything.

Reply:
Nice mantra!
Well, I'll go ahead and remove it.